src/parse/rules/misc_rules.build_defs

```diff
@@ -307,8 +307,8 @@ def system_library(name:str, srcs:list, deps:list=None, hashes:list=None,
 
 
 def remote_file(name:str, url:str|list, hashes:list=None, out:str=None, binary:bool=False,
-                visibility:list=None, licences:list=None, test_only:bool&testonly=False, deps:list=None,
-                exported_deps:list=None, _tag=''):
+                visibility:list=None, licences:list=None, test_only:bool&testonly=False,
+                labels:list=[], deps:list=None, exported_deps:list=None, _tag=''):
```

Comment:
nit: labels:list=None?

Reply:
Thanks Will!

These guys are safe now, it's okay to default them to things like lists.